src/workers/sdk.ts

```diff
@@ -54,7 +58,13 @@ const init = async (
     chainId
   );
   api = new ContractsApi(provider, config);
-  const { cache, startDataSync } = initSyncedCache(api.reader, cachedData);
+  const { cache, startDataSync } = initSyncedCache(
+    api.reader,
+    sdkConfig?.cache,
+    sdkConfig?.batchSize,
```

Comment:
you are passing batch size in two place here - but the first place is actually how many pairs to include in a single multicall call to strategyByPair. Default in the SDK is 100.